Admin/AbstractAdmin.php

```diff
@@ -422,8 +422,6 @@
      */
     protected $securityInformation = array();
 
-    protected $cacheIsGranted = array();
```

Comment:
Would be https://github.com/symfony/symfony/pull/20769/files.

I will add `@deprecated` anyway.